rustfmt.toml

```diff
@@ -0,0 +1 @@
+tab_spaces = 2
```

Comment:
respected this line
https://github.com/smallkirby/RustyCowsay/blob/7597216f02eb7ccefb0f32c3ba2012f2d64f8304/src/main.rs#L6